packages/fxa-settings/src/components/images/index.tsx

```diff
@@ -112,3 +114,21 @@ export const SecurityShieldImage = ({ className, ariaHidden }: ImageProps) => (
     {...{ className, ariaHidden }}
   />
 );
+
+export const RecoveryKeyImage = ({ className, ariaHidden }: ImageProps) => (
+  <PreparedImage
+    ariaLabel="Illustration to represent an account recovery key."
```

Comment:
This is fine, though `Illustration of a key` might be a better descriptor of what the illustration really is?

Reply:
I used the aria-labels that were requested by CX in the Figma doc.